flow/project.py

```diff
@@ -2208,6 +2321,36 @@ def run_operations(self, operations=None, pretend=False, np=None, timeout=None,
                             raise RuntimeError("Unable to parallelize execution due to a pickling "
                                                "error: {}.".format(error))
 
+    @deprecated(deprecated_in="0.11", removed_in="0.13", current_version=__version__)
+    def run_operations(self, operations=None, pretend=False, np=None, timeout=None, progress=False):
+        """Execute the next operations as specified by the project's workflow.
+
+        See also: :meth:`~.run`
+
+        :param operations:
+            The operations to execute (optional).
+        :type operations:
+            Sequence of instances of :class:`._JobOperation`
```

Comment:
```suggestion
            Sequence of instances of :class:`.JobOperation`
```
Public facing methods even with deprecation should reference the public class even if that is also deprecated.